lib/commonmark.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
 var MarkdownIt = require('markdown-it');
-var sanitizer = require('sanitizer');
-var uriPolicy = require('./uriPolicy').uriPolicy;
+var createDOMPurify = require('dompurify');
+var { JSDOM } = require('jsdom');
```

Comment:
The use of `jsdom` must be only done for tests. It must not be present in the implementation.

Reply:
```jsdom``` moved to ```renderHtml``` for tests